sharktank/sharktank/layers/ffn_block.py

```diff
@@ -22,18 +23,29 @@ class FFN(ThetaLayer):
     def __init__(
         self,
         theta: Theta,
+        is_gated: bool = None,
+        activation_fn: Optional[Callable[[AnyTensor], AnyTensor]] = None,
     ):
         super().__init__(theta)
 
-        self.add_module("ffn_gate", LinearLayer(theta("ffn_gate")))
+        self.is_gated = is_gated is None or is_gated
```

Comment:
If default value is boolean, will not need an `or` statement here. Just `self.is_gated = is_gated`

Reply:
Done